kusto/ingest/file_options.go

```diff
@@ -9,57 +9,58 @@ import (
 	"github.com/Azure/azure-kusto-go/kusto/ingest/internal/properties"
 )
 
-type OptionScope uint
+type TypeScope uint
+type ClientScope uint
 
 const (
-	IngestFromFile OptionScope = 1 << iota
-	IngestFromReader
-	IngestFromBlob
-	QueuedIngest
-	StreamingIngest
+	FromFile TypeScope = 1 << iota
+	FromReader
+	FromBlob
+	QueuedClient ClientScope = 1 << iota
+	StreamingClient
 )
 
-var scopesMap = map[OptionScope]string{
-	IngestFromFile:   "IngestFromFile",
-	IngestFromReader: "IngestFromReader",
-	IngestFromBlob:   "IngestFromBlob",
-	QueuedIngest:     "QueuedIngest",
-	StreamingIngest:  "StreamingIngest",
-}
-
 // FileOption is an optional argument to FromFile().
 type FileOption interface {
 	fmt.Stringer
 
-	OptionScopes() OptionScope
+	TypeScopes() TypeScope
```

Comment:
SourceScope since our terminology is SourceOptions etc

Reply:
Done